openpype/hosts/houdini/plugins/publish/validate_review_colorspace.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from openpype.pipeline import (
+    PublishValidationError,
+    OptionalPyblishPluginMixin
+)
+from openpype.pipeline.publish import RepairAction
+from openpype.hosts.houdini.api.action import SelectROPAction
+from openpype.hosts.houdini.api.colorspace import get_default_display_view_colorspace  # noqa
+
+import os
+import hou
+
+
+class SetDefaultViewSpaceAction(RepairAction):
+    label = "Set default view space"
```

Comment:
I wonder whether this would be clearer for the artist if it mentions **colorspace** instead of **space** since it might get confusing with world-space, object-space, 3D-space / 2D-space, or a particular space the view/camera is in or whatever.

```suggestion
    label = "Set default view colorspace"
```